tests/python/frontend/onnx/test_forward.py

```diff
@@ -4165,15 +4165,8 @@ def verify_cumsum(indata, axis, exclusive=0, reverse=0, type="float32"):
     "test_cumsum_2d_axis_0/",
     "test_cumsum_2d_axis_1/",
     "test_cumsum_2d_negative_axis/",
-    "test_dequantizelinear/",
```

Comment:
Do we run these tests on CI?

Reply:
Yes, we're running all of the pre-serialized node tests that ship with ONNX against CPU now, except what's skipped in this list. Working on reducing what we skip, and I'll start enabling GPU soon.